src/main/java/org/nbu/company/model/Company.java

```diff
@@ -1,32 +1,44 @@
 package org.nbu.company.model;
 
+import java.util.Date;
 import java.util.List;
 
-import javax.persistence.CascadeType;
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.Id;
-import javax.persistence.OneToMany;
-import javax.persistence.Table;
+import javax.persistence.*;
 
+import org.nbu.company.client.model.Client;
 import org.nbu.company.employee.model.Employee;
 
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Getter;
+import lombok.Setter;
 import lombok.NoArgsConstructor;
+import org.nbu.company.shipment.Shipment;
 
 @NoArgsConstructor
 @AllArgsConstructor
 @Builder
 @Getter
+@Setter
```

Comment:
No Setter is required because there is a specified default constructor and json deserializer will manage to create the object.